libcontainer/intelrdt/intelrdt.go

```diff
@@ -298,20 +312,26 @@ func isIntelRdtMounted() bool {
 	return true
 }
 
-func parseCpuInfoFile(path string) (bool, bool, error) {
```

Comment:
For the sake of easier review and better git history, can you please split this change into a separate commit?

In other words
 1. "intelrdt: change parseCpuInfoFile to return struct"
 2. "intelrdt: add mbm stats"